src/widgets/radio.jsx

```diff
@@ -19,6 +19,146 @@ var captureScratchpadTouchStart =
 
 var cx = React.addons.classSet;
 
+var Choice = React.createClass({
+    propTypes: {
+        checked: React.PropTypes.bool,
+        classSet: React.PropTypes.shape,
+        clue: React.PropTypes.shape,
+        correct: React.PropTypes.bool,
+        content: React.PropTypes.shape,
+        disabled: React.PropTypes.bool,
+        groupName: React.PropTypes.string,
+        onChecked: React.PropTypes.func,
+        type: React.PropTypes.string
+    },
+
+    getDefaultProps: function() {
+        return {
+            checked: false,
+            classSet: {},
+            correct: false,
+            disabled: false,
+            type: 'radio'
+        };
+    },
+
+    render: function() {
+        this.props.classSet[ApiClassNames.RADIO.OPTION] = true;
+        this.props.classSet[ApiClassNames.RADIO.SELECTED] = this.props.checked;
```

Comment:
Components shouldn't mutate their props (or state) (http://facebook.github.io/react/docs/multiple-components.html and http://facebook.github.io/react/docs/component-api.html have some more thoughts on this).

Instead, we could make a new object here, and pass it in:

    var classSet = _.clone(this.props.classSet);
    classSet[ApiClassNames.RADIO.OPTION] = true;
    classSet[ApiClassNames.RADIO.SELECTED] = this.props.checked;